src/py/flwr/superexec/app.py

```diff
@@ -89,6 +91,10 @@ def _parse_args_run_superexec() -> argparse.ArgumentParser:
         help="The directory for the executor.",
         default=".",
     )
+    parser.add_argument(
+        "--config",
```

Comment:
Should we call this `--executor-config`?

Reply:
Wouldn't it be too verbose for users?

Reply:
I'd prefer it for the sake of clarity / to avoid ambiguity

Reply:
I see, should we also provide an abbreviated option?

Reply:
Yes, we can do that